src/pipeline_config.py

```diff
@@ -284,6 +284,10 @@
                                               },
                       },
 
+    'xgb_preprocessing': {'one_hot_encoder': {'cols': CATEGORICAL_COLUMNS,
+                                              'drop_invariant': False}
```

Comment:
Why don't you drop invariant columns? This options is supposed to drop additional 'Nan' columns if there are no Nans in the (training) data. In our data there are Nans in categorical features only if there are some in training data, so I can't see any obstruction to drop invariant columns.

Reply:
I agree